code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1365,12 +1365,15 @@ var/list/rank_prefix = list(\
 	..()
 
 /mob/living/carbon/human/has_brain()
-	return istype(internal_organs_by_name[BP_BRAIN], /obj/item/organ/internal/brain)
+	if(organ_list_by_process(BP_BRAIN).len)
+		return 1
+	return 0
 
 /mob/living/carbon/human/has_eyes()
-	if(internal_organs_by_name[BP_EYES])
-		var/obj/item/organ/internal/eyes = internal_organs_by_name[BP_EYES]
-		if(eyes && istype(eyes) && !(eyes.status & ORGAN_CUT_AWAY))
+	if(organ_list_by_process(BP_EYES).len)
+		for(var/obj/item/organ/internal/eyes in organ_list_by_process(OP_EYES))
+			if(!(eyes && istype(eyes) && !(eyes.status & ORGAN_CUT_AWAY)))
+				return 0
```

Comment:
it would be nice to replace these ones and zeros with true and false.

Reply:
Fixed.